treewide: bump copyright year to 2022 in generated files

tklauser opened the pull request:
Generated using:
    
    sed -i 's/2017-2021/2017-2022/' Documentation/conf.py hack/custom-boilerplate.go.txt hack/spdx-copyright-header.txt
    make generate-api generate-health-api generate-hubble-api generate-operator-api generate-k8s-api

sayboras reviewed (approved):
LGTM :100: 

contrib/scripts/check-api-code-gen.sh should be run once for operator package.

tklauser committed:
treewide: bump copyright year to 2022 in generated files

Generated using:

    sed -i 's/2017-2021/2017-2022/' Documentation/conf.py hack/custom-boilerplate.go.txt hack/spdx-copyright-header.txt
    make generate-api generate-health-api generate-hubble-api generate-operator-api generate-k8s-api

Signed-off-by: Tobias Klauser <tobias@cilium.io>

tklauser committed:
contrib/scripts: mention generate-operator-api in API check script

Such that the hint message matches the commands which were executed by
the check-api-code-gen.sh script.

Signed-off-by: Tobias Klauser <tobias@cilium.io>
